### PR TITLE
termius 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by EvgeneOskin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/termius.rb [hunk 1]
```diff
@@ -0,0 +1,175 @@
+class Termius < Formula
+  desc "CLI tool for termius.com (aka serverauditor.com)"
+  homepage "https://serverauditor.com"
+
+  stable do
```
Comment (MikeMcQuaid) [inline Formula/termius.rb]: Don't need to use `stable do` when there's just the URL and SHA256 in the block.

File: Formula/termius.rb [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    url "https://github.com/Crystalnix/serverauditor-sshconfig/archive/v1.0.0.tar.gz"
+    sha256 "f0823e52f98ad35c74a4eabac622993df49f6683f64eb3da8850e9368c908171"
+  end
+
+  head do
```
Comment (MikeMcQuaid) [inline Formula/termius.rb]: Don't need to use `head do` when there's just the URL in the block.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
It may also be preferable to use `virtualenv_install_with_resources` in `install`.

**Comment by EvgeneOskin** [comment]
Hi, @MikeMcQuaid. Thank you for review. It was very useful!

I've updated PR.

**Comment by EvgeneOskin** [comment]
Hi @woodruffw. Thank you for your comment. I've found a bug with installation and fixed it. 

I've updated the PR.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**